cmd/oras/root/manifest/index/create.go

```diff
@@ -91,6 +89,10 @@ Example - Create an index and output the index to stdout, auto push will be disa
 			opts.RawReference = refs[0]
 			opts.extraRefs = refs[1:]
 			opts.sources = args[1:]
+			// ignore --pretty when output to a file
+			if opts.outputPath != "" && opts.outputPath != "-" {
+				opts.Pretty.Pretty = false
```

Comment:
In the option handling in command maybe?

Reply:
> In the option handling in command maybe?

It's better to put the reset in content handler, since both `index create` and `index update` can use this code snippet from the handler. @TerryHowe

Reply:
Moved this part to `content.NewManifestIndexCreateHandler`